guides/doc-Installing_Server/master.adoc

```diff
@@ -62,9 +62,12 @@ include::common/assembly_configuring-external-services.adoc[leveloffset=+1]
 
 :numbered!:
 
-ifdef::foreman-el,katello[]
+ifndef::foreman-deb[]
 [appendix]
-include::common/modules/con_troubleshooting.adoc[leveloffset=+1]
```

Comment:
Should this module also be included somewhere?

Reply:
I guess that file/module is redundant now with this PR because of splitting.